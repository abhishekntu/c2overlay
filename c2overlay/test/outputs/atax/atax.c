/****************************************************************/
/*   chebyshev		                                        */
/****************************************************************/

#ifdef TEST
#include <stdio.h>
#endif

#include<math.h>

#ifdef ALGO
void foo()
{
    int A [9];
    int x [3];
    int y [3];
    int i;
    {
        y[0] = (A[0] * (A[0] * x[0] + A[1] * x[1] + A[2] * x[2]) + A[3] * (A[3] * x[0] + A[4] * x[1] + A[5] * x[2]) + A[6] * (A[6] * x[0] + A[7] * x[1] + A[8] * x[2]));
        y[1] = (A[1] * (A[0] * x[0] + A[1] * x[1] + A[2] * x[2]) + A[4] * (A[3] * x[0] + A[4] * x[1] + A[5] * x[2]) + A[7] * (A[6] * x[0] + A[7] * x[1] + A[8] * x[2]));
        y[2] = (A[2] * (A[0] * x[0] + A[1] * x[1] + A[2] * x[2]) + A[5] * (A[3] * x[0] + A[4] * x[1] + A[5] * x[2]) + A[8] * (A[6] * x[0] + A[7] * x[1] + A[8] * x[2]));
    }

}
#endif


#ifdef TEST
int main(void)
{
	int x;
 	int result;
  	foo();
  	return 0;
}
#endif
