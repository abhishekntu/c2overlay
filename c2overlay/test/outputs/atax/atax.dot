digraph graphname {
N49 [color="black", label="store_Imm_1_N74_N49"];
N48 [color="black", label="N37_mul_N37_N48"];
N41 [color="black", label="N50_mul_N50_N41"];
N40 [color="black", label="N53N61N54_add_N53_mul_N61_add_N54_N40"];
N43 [color="black", label="N39_mul_N39_N43"];
N42 [color="black", label="N51_mul_N51_N42"];
N45 [color="black", label="N33N13_mul_N33_add_N13_N45"];
N44 [color="black", label="N32_mul_N32_N44"];
N47 [color="black", label="N35_mul_N35_N47"];
N46 [color="black", label="N34N31_mul_N34_add_N31_N46"];
N29 [color="black", label="N23N38_mul_N23_add_N38_N29"];
N28 [color="black", label="N59_mul_N59_N28"];
N23 [color="black", label="N45N65N16_add_N45_mul_N65_add_N16_N23"];
N22 [color="black", label="N42N56_mul_N42_add_N56_N22"];
N21 [color="black", label="N43_mul_N43_N21"];
N20 [color="black", label="N41N60_mul_N41_add_N60_N20"];
N27 [color="black", label="N14N30N15_add_N14_mul_N30_add_N15_N27"];
N26 [color="black", label="N29_mul_N29_N26"];
N25 [color="black", label="N69N57N68_add_N69_mul_N57_add_N68_N25"];
N24 [color="black", label="N44_mul_N44_N24"];
N8 [color="black", label="load_Imm_3_N5_N8"];
N9 [color="black", label="load_Imm_1_N6_N9"];
N1 [color="black", label="load_Imm_1_N12_N1"];
N2 [color="black", label="load_Imm_0_N8_N2"];
N3 [color="black", label="load_Imm_0_N9_N3"];
N4 [color="black", label="load_Imm_8_N1_N4"];
N5 [color="black", label="load_Imm_5_N2_N5"];
N6 [color="black", label="load_Imm_6_N3_N6"];
N7 [color="black", label="load_Imm_4_N4_N7"];
N12 [color="black", label="load_Imm_2_N11_N12"];
N13 [color="black", label="N55N70_mul_N55_add_N70_N13"];
N10 [color="black", label="load_Imm_7_N7_N10"];
N11 [color="black", label="load_Imm_2_N10_N11"];
N16 [color="black", label="N67N36_mul_N67_add_N36_N16"];
N17 [color="black", label="N66N47_mul_N66_add_N47_N17"];
N14 [color="black", label="N46N27_mul_N46_add_N27_N14"];
N15 [color="black", label="N19N72N18_add_N19_mul_N72_add_N18_N15"];
N18 [color="black", label="N64_mul_N64_N18"];
N19 [color="black", label="N63N17N40_add_N63_mul_N17_add_N40_N19"];
N50 [color="black", label="store_Imm_2_N75_N50"];
N51 [color="black", label="store_Imm_0_N73_N51"];
N38 [color="black", label="N58_mul_N58_N38"];
N39 [color="black", label="N52N62_mul_N52_add_N62_N39"];
N30 [color="black", label="N22_mul_N22_N30"];
N31 [color="black", label="N21N20_mul_N21_add_N20_N31"];
N32 [color="black", label="N26_mul_N26_N32"];
N33 [color="black", label="N25_mul_N25_N33"];
N34 [color="black", label="N24_mul_N24_N34"];
N35 [color="black", label="N49N28_mul_N49_add_N28_N35"];
N36 [color="black", label="N48_mul_N48_N36"];
N37 [color="black", label="N71_mul_N71_N37"];
N48 -> N45;
N41 -> N13;
N40 -> N19;
N43 -> N40;
N42 -> N39;
N45 -> N27;
N44 -> N14;
N47 -> N27;
N46 -> N47;
N29 -> N23;
N28 -> N40;
N23 -> N25;
N22 -> N40;
N21 -> N20;
N20 -> N46;
N27 -> N15;
N26 -> N27;
N25 -> N50;
N24 -> N22;
N8 -> N17;
N8 -> N22;
N8 -> N45;
N8 -> N40;
N9 -> N20;
N9 -> N44;
N9 -> N47;
N9 -> N29;
N1 -> N44;
N1 -> N29;
N1 -> N48;
N1 -> N31;
N1 -> N13;
N1 -> N33;
N1 -> N20;
N1 -> N42;
N1 -> N24;
N2 -> N34;
N2 -> N43;
N2 -> N14;
N2 -> N21;
N3 -> N41;
N3 -> N32;
N3 -> N14;
N3 -> N39;
N3 -> N21;
N3 -> N17;
N3 -> N45;
N3 -> N22;
N3 -> N34;
N4 -> N38;
N4 -> N37;
N4 -> N30;
N4 -> N25;
N5 -> N36;
N5 -> N28;
N5 -> N26;
N5 -> N35;
N6 -> N41;
N6 -> N39;
N6 -> N32;
N6 -> N19;
N7 -> N33;
N7 -> N24;
N7 -> N48;
N7 -> N27;
N12 -> N30;
N12 -> N35;
N12 -> N28;
N12 -> N16;
N12 -> N37;
N12 -> N18;
N12 -> N26;
N12 -> N38;
N12 -> N46;
N13 -> N25;
N10 -> N13;
N10 -> N31;
N10 -> N42;
N10 -> N15;
N11 -> N16;
N11 -> N46;
N11 -> N18;
N11 -> N23;
N16 -> N43;
N17 -> N35;
N14 -> N16;
N15 -> N49;
N18 -> N23;
N19 -> N51;
N38 -> N25;
N39 -> N19;
N30 -> N15;
N31 -> N15;
N32 -> N31;
N33 -> N17;
N34 -> N29;
N35 -> N36;
N36 -> N23;
N37 -> N19;
}
