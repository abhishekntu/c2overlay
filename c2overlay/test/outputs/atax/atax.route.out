Array size: 7 x 7 logic blocks.

Routing:

Net 0 (N8)

SOURCE (4,0)  Pad: 1  
  OPIN (4,0)  Pad: 1  
 CHANX (4,0)  Track: 3  
 CHANY (3,1)  Track: 2  
 CHANY (3,2)  Track: 2  
  IPIN (4,2)  Pin: 3  
  SINK (4,2)  Class: 0  
 CHANY (3,1)  Track: 2  
 CHANX (4,1)  Track: 2  
 CHANX (5,1)  Track: 2  
  IPIN (5,2)  Pin: 2  
  SINK (5,2)  Class: 0  
 CHANY (3,2)  Track: 2  
 CHANY (3,3)  Track: 2  
 CHANY (3,4)  Track: 2  
  IPIN (4,4)  Pin: 3  
  SINK (4,4)  Class: 0  
 CHANY (3,4)  Track: 2  
 CHANY (3,5)  Track: 2  
  IPIN (4,5)  Pin: 3  
  SINK (4,5)  Class: 0  


Net 1 (N9)

SOURCE (0,1)  Pad: 1  
  OPIN (0,1)  Pad: 1  
 CHANY (0,1)  Track: 3  
  IPIN (1,1)  Pin: 3  
  SINK (1,1)  Class: 0  
 CHANY (0,1)  Track: 3  
 CHANX (1,0)  Track: 2  
 CHANX (2,0)  Track: 2  
  IPIN (2,1)  Pin: 2  
  SINK (2,1)  Class: 0  
 CHANX (2,0)  Track: 2  
 CHANY (2,1)  Track: 2  
 CHANY (2,2)  Track: 2  
 CHANY (2,3)  Track: 2  
  IPIN (2,3)  Pin: 1  
  SINK (2,3)  Class: 0  
 CHANY (2,3)  Track: 2  
 CHANX (2,3)  Track: 3  
 CHANX (1,3)  Track: 3  
  IPIN (1,3)  Pin: 0  
  SINK (1,3)  Class: 0  


Net 2 (N1)

SOURCE (2,0)  Pad: 1  
  OPIN (2,0)  Pad: 1  
 CHANX (2,0)  Track: 3  
 CHANY (1,1)  Track: 2  
  IPIN (2,1)  Pin: 3  
  SINK (2,1)  Class: 0  
 CHANY (1,1)  Track: 2  
  IPIN (1,1)  Pin: 1  
  SINK (1,1)  Class: 0  
 CHANY (1,1)  Track: 2  
 CHANY (1,2)  Track: 2  
 CHANY (1,3)  Track: 2  
  IPIN (2,3)  Pin: 3  
  SINK (2,3)  Class: 0  
 CHANY (1,3)  Track: 2  
 CHANX (2,3)  Track: 2  
 CHANX (3,3)  Track: 2  
 CHANX (4,3)  Track: 2  
 CHANX (5,3)  Track: 2  
  IPIN (5,3)  Pin: 0  
  SINK (5,3)  Class: 0  
 CHANX (5,3)  Track: 2  
  IPIN (5,4)  Pin: 2  
  SINK (5,4)  Class: 0  
 CHANX (5,3)  Track: 2  
 CHANX (6,3)  Track: 2  
 CHANX (7,3)  Track: 2  
  IPIN (7,4)  Pin: 2  
  SINK (7,4)  Class: 0  
 CHANX (6,3)  Track: 2  
 CHANY (6,3)  Track: 3  
 CHANY (6,2)  Track: 3  
  IPIN (7,2)  Pin: 3  
  SINK (7,2)  Class: 0  
 CHANY (6,2)  Track: 3  
 CHANY (6,1)  Track: 3  
  IPIN (7,1)  Pin: 3  
  SINK (7,1)  Class: 0  
 CHANX (4,3)  Track: 2  
 CHANY (4,4)  Track: 2  
 CHANY (4,5)  Track: 2  
 CHANY (4,6)  Track: 2  
  IPIN (4,6)  Pin: 1  
  SINK (4,6)  Class: 0  


Net 3 (N2)

SOURCE (3,0)  Pad: 1  
  OPIN (3,0)  Pad: 1  
 CHANX (3,0)  Track: 3  
  IPIN (3,1)  Pin: 2  
  SINK (3,1)  Class: 0  
  OPIN (3,0)  Pad: 1  
 CHANX (3,0)  Track: 2  
 CHANX (4,0)  Track: 2  
  IPIN (4,1)  Pin: 2  
  SINK (4,1)  Class: 0  
  OPIN (3,0)  Pad: 1  
 CHANX (3,0)  Track: 1  
 CHANY (2,1)  Track: 0  
 CHANY (2,2)  Track: 0  
  IPIN (3,2)  Pin: 3  
  SINK (3,2)  Class: 0  
 CHANY (2,2)  Track: 0  
 CHANY (2,3)  Track: 0  
  IPIN (3,3)  Pin: 3  
  SINK (3,3)  Class: 0  


Net 4 (N3)

SOURCE (6,0)  Pad: 1  
  OPIN (6,0)  Pad: 1  
 CHANX (6,0)  Track: 3  
  IPIN (6,1)  Pin: 2  
  SINK (6,1)  Class: 0  
 CHANX (6,0)  Track: 3  
 CHANY (5,1)  Track: 2  
 CHANY (5,2)  Track: 2  
  IPIN (6,2)  Pin: 3  
  SINK (6,2)  Class: 0  
 CHANY (5,2)  Track: 2  
  IPIN (5,2)  Pin: 1  
  SINK (5,2)  Class: 0  
 CHANY (5,2)  Track: 2  
 CHANY (5,3)  Track: 2  
  IPIN (6,3)  Pin: 3  
  SINK (6,3)  Class: 0  
  OPIN (6,0)  Pad: 1  
 CHANX (6,0)  Track: 1  
 CHANY (5,1)  Track: 0  
 CHANX (5,1)  Track: 1  
 CHANX (4,1)  Track: 1  
  IPIN (4,1)  Pin: 0  
  SINK (4,1)  Class: 0  
 CHANX (4,1)  Track: 1  
 CHANY (3,1)  Track: 1  
  IPIN (3,1)  Pin: 1  
  SINK (3,1)  Class: 0  
 CHANY (5,2)  Track: 2  
 CHANX (5,2)  Track: 3  
 CHANX (4,2)  Track: 3  
 CHANX (3,2)  Track: 3  
  IPIN (3,3)  Pin: 2  
  SINK (3,3)  Class: 0  
 CHANY (5,3)  Track: 2  
 CHANX (5,3)  Track: 3  
 CHANX (4,3)  Track: 3  
  IPIN (4,4)  Pin: 2  
  SINK (4,4)  Class: 0  
 CHANY (5,3)  Track: 2  
 CHANY (5,4)  Track: 2  
 CHANX (5,4)  Track: 3  
 CHANX (4,4)  Track: 3  
  IPIN (4,5)  Pin: 2  
  SINK (4,5)  Class: 0  


Net 5 (N4)

SOURCE (8,4)  Pad: 1  
  OPIN (8,4)  Pad: 1  
 CHANY (7,4)  Track: 3  
 CHANX (7,3)  Track: 3  
 CHANX (6,3)  Track: 3  
  IPIN (6,4)  Pin: 2  
  SINK (6,4)  Class: 0  
 CHANX (7,3)  Track: 3  
 CHANY (6,4)  Track: 2  
 CHANY (6,5)  Track: 2  
  IPIN (6,5)  Pin: 1  
  SINK (6,5)  Class: 0  
 CHANY (6,5)  Track: 2  
 CHANX (6,5)  Track: 3  
 CHANX (5,5)  Track: 3  
  IPIN (5,5)  Pin: 0  
  SINK (5,5)  Class: 0  
 CHANY (6,5)  Track: 2  
  IPIN (7,5)  Pin: 3  
  SINK (7,5)  Class: 0  


Net 6 (N5)

SOURCE (0,5)  Pad: 1  
  OPIN (0,5)  Pad: 1  
 CHANY (0,5)  Track: 3  
  IPIN (1,5)  Pin: 3  
  SINK (1,5)  Class: 0  
 CHANY (0,5)  Track: 3  
 CHANX (1,4)  Track: 2  
 CHANX (2,4)  Track: 2  
  IPIN (2,5)  Pin: 2  
  SINK (2,5)  Class: 0  
 CHANX (2,4)  Track: 2  
 CHANX (3,4)  Track: 2  
  IPIN (3,5)  Pin: 2  
  SINK (3,5)  Class: 0  
 CHANX (3,4)  Track: 2  
  IPIN (3,4)  Pin: 0  
  SINK (3,4)  Class: 0  


Net 7 (N6)

SOURCE (8,2)  Pad: 1  
  OPIN (8,2)  Pad: 1  
 CHANY (7,2)  Track: 3  
 CHANX (7,1)  Track: 3  
 CHANX (6,1)  Track: 3  
  IPIN (6,2)  Pin: 2  
  SINK (6,2)  Class: 0  
 CHANX (6,1)  Track: 3  
  IPIN (6,1)  Pin: 0  
  SINK (6,1)  Class: 0  
 CHANX (6,1)  Track: 3  
 CHANY (5,1)  Track: 3  
  IPIN (5,1)  Pin: 1  
  SINK (5,1)  Class: 0  
 CHANX (7,1)  Track: 3  
 CHANY (6,2)  Track: 2  
 CHANY (6,3)  Track: 2  
  IPIN (6,3)  Pin: 1  
  SINK (6,3)  Class: 0  


Net 8 (N7)

SOURCE (4,8)  Pad: 1  
  OPIN (4,8)  Pad: 1  
 CHANX (4,7)  Track: 3  
 CHANY (3,7)  Track: 3  
 CHANY (3,6)  Track: 3  
  IPIN (4,6)  Pin: 3  
  SINK (4,6)  Class: 0  
 CHANY (3,6)  Track: 3  
 CHANY (3,5)  Track: 3  
 CHANY (3,4)  Track: 3  
 CHANY (3,3)  Track: 3  
 CHANX (4,2)  Track: 2  
 CHANY (4,3)  Track: 2  
  IPIN (4,3)  Pin: 1  
  SINK (4,3)  Class: 0  
 CHANY (4,3)  Track: 2  
  IPIN (5,3)  Pin: 3  
  SINK (5,3)  Class: 0  
 CHANY (3,5)  Track: 3  
 CHANX (4,4)  Track: 2  
 CHANX (5,4)  Track: 2  
  IPIN (5,4)  Pin: 0  
  SINK (5,4)  Class: 0  


Net 9 (N12)

SOURCE (0,4)  Pad: 1  
  OPIN (0,4)  Pad: 1  
 CHANY (0,4)  Track: 3  
  IPIN (1,4)  Pin: 3  
  SINK (1,4)  Class: 0  
  OPIN (0,4)  Pad: 1  
 CHANY (0,4)  Track: 0  
 CHANX (1,4)  Track: 0  
  IPIN (1,5)  Pin: 2  
  SINK (1,5)  Class: 0  
 CHANX (1,4)  Track: 0  
 CHANX (2,4)  Track: 0  
 CHANY (2,5)  Track: 0  
  IPIN (3,5)  Pin: 3  
  SINK (3,5)  Class: 0  
 CHANX (2,4)  Track: 0  
 CHANY (2,4)  Track: 1  
  IPIN (3,4)  Pin: 3  
  SINK (3,4)  Class: 0  
 CHANY (2,5)  Track: 0  
 CHANX (3,5)  Track: 0  
 CHANX (4,5)  Track: 0  
 CHANY (4,5)  Track: 1  
  IPIN (5,5)  Pin: 3  
  SINK (5,5)  Class: 0  
 CHANX (4,5)  Track: 0  
 CHANX (5,5)  Track: 0  
 CHANX (6,5)  Track: 0  
  IPIN (6,5)  Pin: 0  
  SINK (6,5)  Class: 0  
 CHANX (5,5)  Track: 0  
 CHANY (5,5)  Track: 1  
 CHANY (5,4)  Track: 1  
  IPIN (6,4)  Pin: 3  
  SINK (6,4)  Class: 0  
 CHANY (0,4)  Track: 3  
 CHANY (0,3)  Track: 3  
 CHANY (0,2)  Track: 3  
  IPIN (1,2)  Pin: 3  
  SINK (1,2)  Class: 0  
 CHANY (2,4)  Track: 1  
 CHANY (2,3)  Track: 1  
 CHANY (2,2)  Track: 1  
  IPIN (2,2)  Pin: 1  
  SINK (2,2)  Class: 0  


Net 10 (N10)

SOURCE (7,0)  Pad: 1  
  OPIN (7,0)  Pad: 1  
 CHANX (7,0)  Track: 3  
  IPIN (7,1)  Pin: 2  
  SINK (7,1)  Class: 0  
 CHANX (7,0)  Track: 3  
 CHANY (6,1)  Track: 2  
 CHANX (7,1)  Track: 2  
  IPIN (7,2)  Pin: 2  
  SINK (7,2)  Class: 0  
 CHANX (7,1)  Track: 2  
 CHANY (7,2)  Track: 2  
 CHANY (7,3)  Track: 2  
  IPIN (7,3)  Pin: 1  
  SINK (7,3)  Class: 0  
 CHANY (7,3)  Track: 2  
 CHANY (7,4)  Track: 2  
  IPIN (7,4)  Pin: 1  
  SINK (7,4)  Class: 0  


Net 11 (N11)

SOURCE (0,3)  Pad: 1  
  OPIN (0,3)  Pad: 1  
 CHANY (0,3)  Track: 2  
 CHANX (1,3)  Track: 2  
  IPIN (1,4)  Pin: 2  
  SINK (1,4)  Class: 0  
 CHANX (1,3)  Track: 2  
 CHANY (1,4)  Track: 2  
  IPIN (2,4)  Pin: 3  
  SINK (2,4)  Class: 0  
 CHANX (1,3)  Track: 2  
 CHANY (1,3)  Track: 3  
 CHANY (1,2)  Track: 3  
  IPIN (2,2)  Pin: 3  
  SINK (2,2)  Class: 0  
 CHANY (1,2)  Track: 3  
  IPIN (1,2)  Pin: 1  
  SINK (1,2)  Class: 0  


Net 12 (N48)

SOURCE (5,4)  Class: 1  
  OPIN (5,4)  Pin: 7  
 CHANY (4,4)  Track: 3  
  IPIN (4,4)  Pin: 1  
  SINK (4,4)  Class: 0  


Net 13 (N41)

SOURCE (6,3)  Class: 1  
  OPIN (6,3)  Pin: 4  
 CHANX (6,3)  Track: 0  
 CHANY (6,4)  Track: 0  
  IPIN (7,4)  Pin: 3  
  SINK (7,4)  Class: 0  


Net 14 (N40)

SOURCE (4,2)  Class: 1  
  OPIN (4,2)  Pin: 5  
 CHANY (4,2)  Track: 1  
 CHANY (4,1)  Track: 1  
  IPIN (5,1)  Pin: 3  
  SINK (5,1)  Class: 0  


Net 15 (N43)

SOURCE (3,2)  Class: 1  
  OPIN (3,2)  Pin: 5  
 CHANY (3,2)  Track: 1  
 CHANX (4,1)  Track: 0  
  IPIN (4,2)  Pin: 2  
  SINK (4,2)  Class: 0  


Net 16 (N42)

SOURCE (7,1)  Class: 1  
  OPIN (7,1)  Pin: 7  
 CHANY (6,1)  Track: 1  
  IPIN (6,1)  Pin: 1  
  SINK (6,1)  Class: 0  


Net 17 (N45)

SOURCE (4,4)  Class: 1  
  OPIN (4,4)  Pin: 6  
 CHANX (4,3)  Track: 1  
  IPIN (4,3)  Pin: 0  
  SINK (4,3)  Class: 0  


Net 18 (N44)

SOURCE (2,1)  Class: 1  
  OPIN (2,1)  Pin: 5  
 CHANY (2,1)  Track: 1  
  IPIN (3,1)  Pin: 3  
  SINK (3,1)  Class: 0  


Net 19 (N47)

SOURCE (1,3)  Class: 1  
  OPIN (1,3)  Pin: 5  
 CHANY (1,3)  Track: 1  
 CHANX (2,2)  Track: 0  
 CHANX (3,2)  Track: 0  
 CHANX (4,2)  Track: 0  
  IPIN (4,3)  Pin: 2  
  SINK (4,3)  Class: 0  


Net 20 (N46)

SOURCE (1,2)  Class: 1  
  OPIN (1,2)  Pin: 4  
 CHANX (1,2)  Track: 3  
  IPIN (1,3)  Pin: 2  
  SINK (1,3)  Class: 0  


Net 21 (N29)

SOURCE (2,3)  Class: 1  
  OPIN (2,3)  Pin: 4  
 CHANX (2,3)  Track: 0  
  IPIN (2,4)  Pin: 2  
  SINK (2,4)  Class: 0  


Net 22 (N28)

SOURCE (1,5)  Class: 1  
  OPIN (1,5)  Pin: 5  
 CHANY (1,5)  Track: 0  
 CHANX (2,5)  Track: 0  
 CHANY (2,5)  Track: 1  
 CHANX (3,4)  Track: 0  
 CHANY (3,4)  Track: 1  
 CHANX (4,3)  Track: 0  
 CHANY (4,3)  Track: 1  
 CHANX (4,2)  Track: 1  
  IPIN (4,2)  Pin: 0  
  SINK (4,2)  Class: 0  


Net 23 (N23)

SOURCE (2,4)  Class: 1  
  OPIN (2,4)  Pin: 5  
 CHANY (2,4)  Track: 2  
 CHANY (2,5)  Track: 2  
 CHANX (3,5)  Track: 2  
 CHANX (4,5)  Track: 2  
 CHANX (5,5)  Track: 2  
 CHANX (6,5)  Track: 2  
 CHANX (7,5)  Track: 2  
  IPIN (7,5)  Pin: 0  
  SINK (7,5)  Class: 0  


Net 24 (N22)

SOURCE (5,2)  Class: 1  
  OPIN (5,2)  Pin: 7  
 CHANY (4,2)  Track: 3  
  IPIN (4,2)  Pin: 1  
  SINK (4,2)  Class: 0  


Net 25 (N21)

SOURCE (4,1)  Class: 1  
  OPIN (4,1)  Pin: 7  
 CHANY (3,1)  Track: 0  
 CHANX (3,1)  Track: 1  
 CHANX (2,1)  Track: 1  
 CHANX (1,1)  Track: 1  
  IPIN (1,1)  Pin: 0  
  SINK (1,1)  Class: 0  


Net 26 (N20)

SOURCE (1,1)  Class: 1  
  OPIN (1,1)  Pin: 4  
 CHANX (1,1)  Track: 3  
  IPIN (1,2)  Pin: 2  
  SINK (1,2)  Class: 0  


Net 27 (N27)

SOURCE (4,3)  Class: 1  
  OPIN (4,3)  Pin: 5  
 CHANY (4,3)  Track: 0  
 CHANX (5,3)  Track: 0  
 CHANY (5,3)  Track: 1  
 CHANX (6,2)  Track: 0  
 CHANX (7,2)  Track: 0  
  IPIN (7,3)  Pin: 2  
  SINK (7,3)  Class: 0  


Net 28 (N26)

SOURCE (3,4)  Class: 1  
  OPIN (3,4)  Pin: 6  
 CHANX (3,3)  Track: 0  
 CHANY (3,3)  Track: 1  
  IPIN (4,3)  Pin: 3  
  SINK (4,3)  Class: 0  


Net 29 (N50)

SOURCE (7,5)  Class: 1  
  OPIN (7,5)  Pin: 5  
 CHANY (7,5)  Track: 2  
  IPIN (8,5)  Pad: 0  
  SINK (8,5)  Pad: 0  


Net 30 (N24)

SOURCE (5,3)  Class: 1  
  OPIN (5,3)  Pin: 6  
 CHANX (5,2)  Track: 2  
  IPIN (5,2)  Pin: 0  
  SINK (5,2)  Class: 0  


Net 31 (N13)

SOURCE (7,4)  Class: 1  
  OPIN (7,4)  Pin: 4  
 CHANX (7,4)  Track: 3  
  IPIN (7,5)  Pin: 2  
  SINK (7,5)  Class: 0  


Net 32 (N16)

SOURCE (2,2)  Class: 1  
  OPIN (2,2)  Pin: 5  
 CHANY (2,2)  Track: 3  
 CHANX (3,1)  Track: 2  
  IPIN (3,2)  Pin: 2  
  SINK (3,2)  Class: 0  


Net 33 (N17)

SOURCE (4,5)  Class: 1  
  OPIN (4,5)  Pin: 7  
 CHANY (3,5)  Track: 1  
  IPIN (3,5)  Pin: 1  
  SINK (3,5)  Class: 0  


Net 34 (N14)

SOURCE (3,1)  Class: 1  
  OPIN (3,1)  Pin: 4  
 CHANX (3,1)  Track: 3  
 CHANX (2,1)  Track: 3  
  IPIN (2,2)  Pin: 2  
  SINK (2,2)  Class: 0  


Net 35 (N49)

SOURCE (7,3)  Class: 1  
  OPIN (7,3)  Pin: 5  
 CHANY (7,3)  Track: 1  
  IPIN (8,3)  Pad: 0  
  SINK (8,3)  Pad: 0  


Net 36 (N18)

SOURCE (1,4)  Class: 1  
  OPIN (1,4)  Pin: 4  
 CHANX (1,4)  Track: 3  
 CHANY (0,5)  Track: 2  
 CHANX (1,5)  Track: 2  
 CHANX (2,5)  Track: 2  
 CHANY (2,5)  Track: 3  
 CHANY (2,4)  Track: 3  
  IPIN (2,4)  Pin: 1  
  SINK (2,4)  Class: 0  


Net 37 (N51)

SOURCE (5,1)  Class: 1  
  OPIN (5,1)  Pin: 6  
 CHANX (5,0)  Track: 3  
  IPIN (5,0)  Pad: 0  
  SINK (5,0)  Pad: 0  


Net 38 (N38)

SOURCE (6,5)  Class: 1  
  OPIN (6,5)  Pin: 5  
 CHANY (6,5)  Track: 1  
 CHANX (7,4)  Track: 0  
 CHANY (7,5)  Track: 0  
  IPIN (7,5)  Pin: 1  
  SINK (7,5)  Class: 0  


Net 39 (N39)

SOURCE (6,1)  Class: 1  
  OPIN (6,1)  Pin: 7  
 CHANY (5,1)  Track: 1  
 CHANX (5,0)  Track: 1  
  IPIN (5,1)  Pin: 2  
  SINK (5,1)  Class: 0  


Net 40 (N30)

SOURCE (6,4)  Class: 1  
  OPIN (6,4)  Pin: 5  
 CHANY (6,4)  Track: 1  
 CHANY (6,3)  Track: 1  
  IPIN (7,3)  Pin: 3  
  SINK (7,3)  Class: 0  


Net 41 (N31)

SOURCE (7,2)  Class: 1  
  OPIN (7,2)  Pin: 4  
 CHANX (7,2)  Track: 1  
 CHANY (6,3)  Track: 0  
 CHANX (7,3)  Track: 0  
  IPIN (7,3)  Pin: 0  
  SINK (7,3)  Class: 0  


Net 42 (N32)

SOURCE (6,2)  Class: 1  
  OPIN (6,2)  Pin: 4  
 CHANX (6,2)  Track: 2  
 CHANX (7,2)  Track: 2  
  IPIN (7,2)  Pin: 0  
  SINK (7,2)  Class: 0  


Net 43 (N33)

SOURCE (4,6)  Class: 1  
  OPIN (4,6)  Pin: 6  
 CHANX (4,5)  Track: 3  
  IPIN (4,5)  Pin: 0  
  SINK (4,5)  Class: 0  


Net 44 (N34)

SOURCE (3,3)  Class: 1  
  OPIN (3,3)  Pin: 7  
 CHANY (2,3)  Track: 3  
 CHANX (2,2)  Track: 3  
  IPIN (2,3)  Pin: 2  
  SINK (2,3)  Class: 0  


Net 45 (N35)

SOURCE (3,5)  Class: 1  
  OPIN (3,5)  Pin: 4  
 CHANX (3,5)  Track: 3  
 CHANX (2,5)  Track: 3  
  IPIN (2,5)  Pin: 0  
  SINK (2,5)  Class: 0  


Net 46 (N36)

SOURCE (2,5)  Class: 1  
  OPIN (2,5)  Pin: 6  
 CHANX (2,4)  Track: 3  
  IPIN (2,4)  Pin: 0  
  SINK (2,4)  Class: 0  


Net 47 (N37)

SOURCE (5,5)  Class: 1  
  OPIN (5,5)  Pin: 5  
 CHANY (5,5)  Track: 3  
 CHANY (5,4)  Track: 3  
 CHANY (5,3)  Track: 3  
 CHANY (5,2)  Track: 3  
 CHANX (5,1)  Track: 3  
  IPIN (5,1)  Pin: 0  
  SINK (5,1)  Class: 0  
